### ISSUE TITLE
Font Problem

### ISSUE THREAD

**Issue by henriquedrumond** [UXDivers/Grial-UI-Kit-Support#604]
I am using the line in my project - local:GrialIconsFont.Star. The code line is ok and in my screen I can see one Star.

So, when I use the line local:GrialIconsFont.HelpCircle it's not ok. The symbol HelpCircle has in GrialIconsFont class.

Anybody can help me please!?

Tks a lot.

**Comment by jpbrocca** [comment]
Hi @henriquedrumond,

Could you please add more details about your issue? 

Is this happening on  Grial Starter solution or the full Solution?
If you are adding Grial to an existing project you need to make sure you have the right version of the font file (.ttf).

Best 

JP

**Comment by henriquedrumond** [comment]
I doing is copy and paste the part of code in my XAML from Grial Full Example. After paste I changed the icon in the line. When I run my app, I see only ? icon. I read the orientation from Grial Support, but I not changed the shape. I doing a simple copy/paste from functional line.

**Comment by jpbrocca** [comment]
I am not sure I understand your comments, but it seems you are adding XAML stuff from Grial Full solution to your own project, right?

If that is the case, then, make sure to copy the font file from the downloaded solution and paste it on your iOS and Android projects.

If not, then please add more details.

Thanks,
JP

**Comment by henriquedrumond** [comment]
No. I download from Grial ADM Site the Grial Starter and I am changing conform my especifications. When I do copy/paste the line and changed the icon, it happens.

**Comment by jpbrocca** [comment]
Hi Henrique, thanks for the details.

So, I went to Grial Web Admin and downloaded a Grial Starter solution with just one page.
I opened the downloaded solution in Visual Studio, replaced the XAML content with just a ```<Label>``` form the same page. Then I replaced the icon with yours and made font size bigger and this is what I get:

![image](https://user-images.githubusercontent.com/15996999/61820143-d52f0a00-ae2a-11e9-9b2e-9f5688262d9d.png)
